test/suite/wttest.py

```diff
@@ -341,6 +341,10 @@ def initialFileName(self, name):
     def getTimestamp(self):
         return self.platform_api.getTimestamp()
 
+    # Return the tier share percent for this testcase, or None if there is none.
```

Comment:
The `None` in the comment should be `0`?

Reply:
Fixed